checker.cpp

```diff
@@ -1,46 +1,70 @@
-#include <iostream>
-#include <assert.h>
-#include <unistd.h>
+#include "checker.h"
+
 using namespace std;
 
-int vitalsOk(float temperature, float pulseRate, float spo2) {
-  if(temperature > 102 || temperature < 95) {
-    cout << "Temperature critical!\n";
-    for (int i = 0; i < 6; i++)
-    {
-      cout << "\r* " << flush;
-      sleep(1);
-      cout << "\r *" << flush;
-      sleep(1);
+int VitalsMonitor::vitalsOk(float temperature, float pulseRate, float spo2) {
+    int status = VitalsMonitor::bodyTemperatureOk(temperature);
+    if (0 != status) {
+        status = VitalsMonitor::pulseRateOk(pulseRate);
+        if(0 != status) {
+            status = VitalsMonitor::oxygenSaturationOk(spo2);
+        }
+    }
+    return status;
```

Comment:
You could also use a boolean expression like
`return VitalsMonitor::bodyTemperatureOk(..) && VitalsMonitor::pulseRateOk(..) && VitalsMonitor::oxygenSaturationOk()`

Reply:
Thank you. I adopted this changes.